doc/api/errors.md

```diff
@@ -573,6 +573,9 @@ program. For a comprehensive list, see the [`errno`(3) man page][].
 - `ENOTEMPTY` (Directory not empty): A directory with entries was the target
   of an operation that requires an empty directory — usually [`fs.unlink`][].
 
+- `ENOTFOUND` (DNS lookup failed): Indicates a DNS failure of either
+  `UV_EAI_NODATA` or `UV_EAI_NONAME`. This is not a standard POSIX error.
```

Comment:
There's no real context here for the `UV_` prefixes. Perhaps just use the [standard](https://tools.ietf.org/html/rfc2553#page-27) `EAI_NODATA` and `EAI_NONAME`?